CHANGELOG.md

```diff
@@ -3,7 +3,10 @@ Changes from v1.4 to v1.5
 
 API Changes
 -----------
-
+- `drawImage()` now accepts `exptime` and `area` keywords to indicate the image
+  exposure time and telescope collecting area.  (#789)
+- dimensions of `SED` changed from [photons/wavelength-interval] to
```

Comment:
We don't want to write long discussions in the CHANGELOG, but I think it would be worth ~2 sentences about the consequences of these things - in particular, the distinction between SED'ed and non-SED'ed chromatic objects, and what they represent.